### DISCUSSION TITLE
Server Location

### DISCUSSION THREAD

**Discussion by markus-gx** [swellstores/community#78] [Help] [answered]
Hi y'all; 

Is it possible to move the server location of certain stores to a different data centre?  For example, if you are on AWS, to eu-central-1 to maintain GDPR compliance? (Not a deal breaker right now, but got asked this by a potential client where the data is stored). Furthermore it would probably increase the server response time.

**Comment by swellmike** [comment, marked as answer]
Currently, all of our servers are hosted in US zones.  There are plans for supporting EU hosting and GDPR, but this is not possible today.

**Reply by JoeMatkin** [reply]
Surely this is something that should be stated somewhere before developers such as myself spend weeks creating a site using Swell?

I also had a call with some people from Swell and they knew we where from the UK and not once was it mentioned that there are only servers in US. This causing slow API responses and lack of GDPR compliance.

**Reply by ericingram** [reply]
Good point. We should make this more clear in calls and communication to people that are investigating the platform.

Our goal is to eliminate this issue of course. Will be very excited to announce our solution in the future.

**Reply by JoeMatkin** [reply]
Any updates on when servers near or in the UK will be used?

thanks

**Reply by johnrisby** [reply]
@ericingram is Swell certified under the EU/US Data Privacy Framework and the UK and Swiss extensions to it?

**Reply by JoeMatkin** [reply]
Please let use know about this, I'm about to start another project and whetherd I use Swell or not will depend on when there will be server in the UK or near.

**Reply by josias-hosang** [reply]
We are interested too in server locations in EU being GDPR/Swiss compliant and for improved performance since our customers shop location is in Switzerland. Thanks for keeping us updated.

**Reply by markus-gx** [reply]
HI @ericingram - every now and then I am looking at this Discussion post to see if something got updated. Is there any update on this particular topic? 
Would be great - since, to be honest, swell is still the coolest way since I tried it the first time. In the meantime we already published Shopware (Headless) and MedusaJS E-Commerce projects sadly not as satisfying and intuitiv as swell was/is and therefore it is even sadder to have the server location as a dealbreaker. 
Thanks - for an answer.

**Reply by JoeMatkin** [reply]
Hi, also still waiting on this.

It's been about a year now and this is a huge problem for us and I have made many projects since but avoided swell because of the server locations.

Is there an ETA?

**Reply by ericingram** [reply]
We decided to pause this effort for now as we're focused on launching the new Storefront editor. We'll get back to it again in the future I'm sure.

**Reply by JelleGroenendal** [reply]
Any news on EU based servers?

**Comment by genox** [comment]
Just chiming in to let you know that EU hosting would also be something we would like to see, for data protection and for performance reasons.
